test/framework/go.mod

```diff
@@ -7,8 +7,15 @@ require (
 	github.com/onsi/gomega v1.7.1
 	github.com/pkg/errors v0.8.1
 	k8s.io/api v0.0.0-20191121015604-11707872ac1c
+	k8s.io/apiextensions-apiserver v0.0.0-20190918201827-3de75813f604 // indirect
 	k8s.io/apimachinery v0.0.0-20191121015412-41065c7a8c2a
-	k8s.io/client-go v0.0.0-20190918160344-1fbdaa4c8d90
+	k8s.io/client-go v11.0.0+incompatible
 	sigs.k8s.io/cluster-api v0.2.6-0.20191216153338-db9baf9ade0c
 	sigs.k8s.io/controller-runtime v0.4.0
 )
+
+replace (
+	k8s.io/client-go => k8s.io/client-go v0.0.0-20190918160344-1fbdaa4c8d90
+	sigs.k8s.io/cluster-api => ../..
+	sigs.k8s.io/kind => sigs.k8s.io/kind v0.5.1
```

Comment:
We've been supporting kind > 0.6.x

Reply:
Yup...I don't think we need this kind replace directive anymore. I had added it there for other reasons.